website/blog/2020-02-02-upgrade-support.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Introducing Upgrade-Support
+author: Lucas Bento
+authorTitle: React Native Community
+authorURL: https://twitter.com/lbentosilva
+authorImageURL: https://avatars3.githubusercontent.com/u/6207220?s=460&v=4
+authorTwitter: lbentosilva
+category: announcements
+---
+
+After a ton of helpful feedback and discussion by the React Native community, we are eager to announce [Upgrade-Support](https://github.com/react-native-community/upgrade-support), a central place to request and give help when upgrading your app. We welcome every React Native developer to post their issues as well as their solutions to problems encountered when upgrading their applications.
+
+This initiative is part of the React Native Community efforts to minimize the work on upgrading your React Native applications.
+
+## How it works
```

Comment:
One thing that I would love to see in this post is an explanation for _why_ upgrades are so difficult. That would help to frame why this tool is necessary and so great to begin with.

As I've become more familiar with the problem space, I think this problem is mostly unavoidable -- native builds just have conflicts which need resolved. If we explain why that's the case, then we can present this tool as a best case scenario instead of a stop gap.